homeassistant/helpers/config_validation.py

```diff
@@ -203,12 +203,9 @@ def boolean(value: Any) -> bool:
     raise vol.Invalid(f"invalid boolean value {value}")
 
 
-_WS = re.compile("\\s*")
-
-
 def whitespace(value: Any) -> str:
     """Validate result contains only whitespace."""
-    if isinstance(value, str) and _WS.fullmatch(value):
+    if isinstance(value, str) and value.isspace():
```

Comment:
Similar considerations here as with `isdigit` -- I'm not sure if we want to accept _all_ kinds of whitespace here, or should it be limited to ASCII whitespace only, or even only a subset of that.